README.md

```diff
@@ -13,22 +13,29 @@ located.
 const path = require('path');
 
 const nconf = require('nconf');
-require('./lib/nconfetti');
 
-const path = path.resolve(__dirname, 'tests/configs/env');
+require('nconfetti');
 
 nconf.argv()
   .env('__');
 
-nconf.use('Nconfetti', {path: path, env: process.env.NODE_ENV});
+nconf.use('Nconfetti', {path: path.resolve(__dirname, '../tests/configs/without_env')});
 
-console.log(nconf.get('development_config:development_config'));
+console.log(nconf.get('simple_config:entry'));
+```
 
+## Try a sample
 
-```
+- see sample/simple.js
+- runit with
+  - npm install
+  - node samples/simple.js
+ - have fun
```

Comment:
Vorschlag: 

```md
1. navigate to sample/simple
2. run it with

   `npm install`
   `node samples/simple.js`

3. have fun
```

--- 

so sähe das aus:


1. navigate to sample/simple
2. run it with

   `npm install`
   `node samples/simple.js`

3. have fun

--- 

*Fragen*

Geht das wirklich so?

Wo wird `npm install` ausgeführt?